openshift/ci-operator/generate-ci-config.sh

```diff
@@ -1,18 +1,24 @@
 #!/bin/bash
 
 branch=${1-'knative-v0.6.0'}
-openshift=${2-'4.2'}
+openshift=${2-'4.3'}
+promotion_disabled=${3-false}
 
 cat <<EOF
 tag_specification:
+  cluster: https://api.ci.openshift.org
   name: '$openshift'
   namespace: ocp
 promotion:
+  additional_images:
+    knative-eventing-src: src
+  disabled: $promotion_disabled
   cluster: https://api.ci.openshift.org
   namespace: openshift
   name: $branch
 base_images:
   base:
+    cluster: https://api.ci.openshift.org
```

Comment:
I believe the cluster stanza should not be there since a few days ago.

Reply:
It yelled at me, yesterday

Reply:
Oh, you are right -> https://github.com/openshift/release/blob/master/ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.14.2__4.4.yaml#L1-L6